scripts/i18n-push.sh

```diff
@@ -21,9 +21,12 @@
 
 cd `dirname $BASH_SOURCE` && cd ..
 
+# Note we only extract the English language strings so they can be
+# sent to Transifex. All other language strings will be pulled
+# down from Transifex and so don't need to be generated.
 echo "Extracting i18n strings..."
-python manage.py makemessages --all
-python manage.py makemessages --all -d djangojs
+python manage.py makemessages -l en --all
+python manage.py makemessages -l en --all -d djangojs
```

Comment:
The default domain for makemessages is 'django' which covers *.py, *.html and *.txt. So the line before this one captures everything but the JS strings.